app/dashboard/views/fido_setup.py

```diff
@@ -0,0 +1,87 @@
+import uuid
+import json
+import secrets
+import webauthn
+from app.config import URL as SITE_URL
```

Comment:
I think `URL` is not a clear enough variable name. It can refer to any arbitrary URL. So I want to make sure I and other maintainers don't get confused about it.

However the `URL` is a naming convention in this project, so I'm fine if you wish me to change it to `URL`

Reply:
No longer valid due to https://github.com/simple-login/app/pull/159#discussion_r421343108